Source/WTF/wtf/text/StringImpl.h

```diff
@@ -1152,6 +1156,36 @@ inline void StringImpl::copyCharacters(DestinationCharacterType* destination, co
         }
         memcpy(destination, source, numCharacters * sizeof(DestinationCharacterType));
     } else {
+#if CPU(ARM64)
+        // SIMD Upconvert.
+        if constexpr (std::is_same_v<SourceCharacterType, LChar> && std::is_same_v<DestinationCharacterType, UChar>) {
+            const auto* const end = destination + numCharacters;
+            constexpr uintptr_t memoryAccessSize = 64;
+
+            if (numCharacters >= memoryAccessSize) {
+                constexpr uintptr_t memoryAccessMask = memoryAccessSize - 1;
+                const auto* const simdEnd = destination + (numCharacters & ~memoryAccessMask);
+                const uint8x16_t zeros = vdupq_n_u8(0);
```

Comment:
Not sure the `const` makes this easier to read.

Reply:
Dropped.